torch/_inductor/exc.py

```diff
@@ -54,7 +54,9 @@ def __init__(self, target, args, kwargs) -> None:
 
 
 class LoweringException(OperatorIssue):
-    def __init__(self, exc: Exception, target, args, kwargs) -> None:
+    def __init__(
+        self, exc: Exception, target: Any, args: list[Any], kwargs: dict[str, Any]
```

Comment:
Just a warning, there is some old school parts of the JIT codebase that desn't like the need `list[...]` syntax and prefers the typing.List objects